server/schema/contests/createContest.js

```diff
@@ -0,0 +1,77 @@
+const schema = {
+  required: [
+    'contest_name',
+    'creator',
+    'show_leaderboard',
+    'public',
+    'confidential',
+    'start_time',
+    'end_time',
+  ],
+  properties: {
+    contest_name: {
+      type: 'string',
+      minLength: 4,
+    },
+    creator: {
+      type: 'string',
+      minLength: 4,
+    },
+    show_leaderboard: {
+      type: 'boolean',
+    },
+    public: {
+      type: 'boolean',
+    },
+    confidential: {
+      type: 'boolean',
+    },
+    start_time: {
+      type: 'string',
```

Comment:
The date sent would be a JS object created using `new Date()`
```suggestion
      type: 'object',
```